check/testdata/nil_constant_crash.go

```diff
@@ -0,0 +1,25 @@
+package foo
```

Comment:
Put this test with the others in `values.go`, and the calls in the same func as all the others.